### PR TITLE
[Term Entry] Kotlin Datetime .Month() #3813

### PR DESCRIPTION
**Pull request by philipphdt**
### Description

- Added the Month() method for Kotlin
- Closes Issue #3813 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/datetime/terms/Month/Month.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+Title: 'month'
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/datetime/terms/Month/Month.md:2]:
```suggestion
Title: '.Month'
```

File: content/kotlin/concepts/datetime/terms/Month/Month.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Tags:\
+  - 'DateTime'
+  - 'Methods'
+  - 'Kotlin'
+CatalogContent:
+  - 'learn-kotlin'
+---
+
+
+The **.Month()** method in Kotlin is used to retrieve the month from a given date object. This method is essential when working with date and time in Kotlin, allowing developers to extract and manipulate the month component of a date. It's particularly useful in scenarios where the month needs to be displayed, compared, or used in calculations. In Kotlin's date-time API, the month is typically represented as an integer, where 1 corresponds to January and 12 to December.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/datetime/terms/Month/Month.md:16]:
```suggestion
The **`.Month()`** method in Kotlin is used to retrieve the month from a given date object. This method is essential when working with `date` and `time` in Kotlin, allowing developers to extract and manipulate the month component of a date. It's particularly useful in scenarios where the month needs to be displayed, compared, or used in calculations. In Kotlin's date-time API, the month is typically represented as an integer, where `1` corresponds to `January` and `12` to `December`.
```

File: content/kotlin/concepts/datetime/terms/Month/Month.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## Syntax
+
+```kotlin
+val month: Int = date.Month()
+```
+
+- **date**: An instance of a date object from which you want to extract the month.
+- **month**: This is an integer representing the month extracted from the date object. Typically, in Kotlin, months are represented as integers ranging from 1 (January) to 12 (December).
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/datetime/terms/Month/Month.md:25]:
```suggestion
- *date*: An instance of a date object from which you want to extract the month.
- *month*: This is an integer representing the month extracted from the date object. Typically, in Kotlin, months are represented as integers ranging from 1 (January) to 12 (December).
```

File: content/kotlin/concepts/datetime/terms/Month/Month.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+```kotlin
+import java.time.LocalDate
+
+fun main() {
+    val currentDate = LocalDate.now() // Let's assume the current date is 2023-11-01
+    val month = currentDate.monthValue
+
+    println("The current month is: $month") // Output will be: The current month is: 11
+}
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/datetime/terms/Month/Month.md:37]:
```suggestion
import java.time.LocalDate

fun main() {
    val currentDate = LocalDate.now() 
    val month = currentDate.monthValue

    println("The current month is: $month")
}
```

File: content/kotlin/concepts/datetime/terms/Month/Month.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+```
+
+In this example, `LocalDate.now()` is used to get the current date. The `.monthValue` property (which serves a similar purpose to a `.Month()` method) of the `LocalDate` object is then called to retrieve the month from the current date. The month is then printed to the console. Assuming the current date is November 1, 2023, the output will be `11`.
+
+**Note**: Kotlin's standard library doesn't directly provide a `.Month()` method. Instead, the standard way to get the month from a date in Kotlin is by using Java's `java.time.LocalDate` and its `monthValue` property as shown in the example. Adjustments might be needed based on the specific libraries and versions in use.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/datetime/terms/Month/Month.md:42]:
```suggestion
> **Note**: Kotlin's standard library doesn't directly provide a `.Month()` method. Instead, the standard way to get the month from a date in Kotlin is by using Java's `java.time.LocalDate` and its monthValue` property as shown in the example. Adjustments might be needed based on the specific libraries and versions in use.
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
I have request few changes please commit them so we can move forward to 2nd review.

File: content/kotlin/concepts/datetime/terms/Month/Month.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
+---
+Title: 'month'
+Description: 'Retrieves the month from a Kotlin DateTime object as an integer.'
+Subjects:
+  - 'Kotlin'
+  - 'DateTime'
+Tags:\
+  - 'DateTime'
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/datetime/terms/Month/Month.md:8]:
```suggestion
  - 'Date'
```
Date time doesn't exist in the tags.md file

File: content/kotlin/concepts/datetime/terms/Month/Month.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  - 'Methods'
+  - 'Kotlin'
+CatalogContent:
+  - 'learn-kotlin'
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/datetime/terms/Month/Month.md:12]:
```suggestion
  - 'learn-kotlin'
  - 'paths/computer-science'
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]

**Comment by philipphdt** [comment]
@dakshdeepHERE Hi, thanks for your inputs! I hope everything looks good now.

File: content/kotlin/concepts/datetime/terms/Month/Month.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+fun main() {
+    val currentDate = LocalDate.now() 
+    val month = currentDate.monthValue
+
+    println("The current month is: $month")
+}
+```
+
+In this example, `LocalDate.now()` is used to get the current date. The `.monthValue` property (which serves a similar purpose to a `.Month()` method) of the `LocalDate` object is then called to retrieve the month from the current date. The month is then printed to the console. Assuming the current date is November 1, 2023, the output will be `11`.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/datetime/terms/Month/Month.md:41]: Remove this sentence from here and move it above the example code snippet as an introductory sentence. Also, remove the last output part and add the output in its own block. Like this:
```shell
then your output

**Review by dakshdeepHERE — COMMENTED** [review]
I have comment from my side please work on it too

File: content/kotlin/concepts/datetime/terms/Month/Month.md (hunk 8 above)
Comment (dakshdeepHERE) [reply content/kotlin/concepts/datetime/terms/Month/Month.md:41]: image
This is how to make output block 👆🏻

**Comment by philipphdt** [comment]
@dakshdeepHERE Hi, I just added your inputs.

**Review by dakshdeepHERE — APPROVED** [review]
Looks good for 2nd review

File: content/kotlin/concepts/datetime/terms/Month/Month.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
+---
+Title: '.Month'
+Description: 'Retrieves the month from a Kotlin DateTime object as an integer.'
+Subjects:
+  - 'Kotlin'
+  - 'DateTime'
+Tags:\
```
Comment (SSwiniarski) [inline content/kotlin/concepts/datetime/terms/Month/Month.md:7]:
```suggestion
Tags:
```

File: content/kotlin/concepts/datetime/terms/Month/Month.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+  - 'Date'
+  - 'Methods'
+  - 'Kotlin'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
```
Comment (SSwiniarski) [inline content/kotlin/concepts/datetime/terms/Month/Month.md:15]:
```suggestion
```
Remove extra blank line.

File: content/kotlin/concepts/datetime/terms/Month/Month.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+import java.time.LocalDate
+
+fun main() {
+    val currentDate = LocalDate.now() 
+    val month = currentDate.monthValue
+
+    println("The current month is: $month")
+}
+```
+Assuming the current date is November 1, 2023, the output will be:
```
Comment (SSwiniarski) [inline content/kotlin/concepts/datetime/terms/Month/Month.md:42]:
```suggestion

Assuming the current date is November 1, 2023, the output will be:
```
Blank line between code block and next paragraph.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@philipphdt I did a second review and left some comments.

**Comment by SSwiniarski** [comment]
@KTom101, @caupolicandiaz There seems to be a mismatch here. Most of the Kotlin datetime terms seem to refer to the `kotlinx.datetime` library. This doesn't. How do we want to deal with this?

**Comment by KTom101** [comment]
@SSwiniarski @philipphdt  I'm ok with keeping this. However, it would be a good idea to move the blockquote note at the bottom to before the syntax section. That way it's toward the top and the reader is aware of this before continuing on.

**Comment by caupolicandiaz** [comment]
@SSwiniarski I've made some outstanding edits to the metadata and updated elements of the body to better align this content with the other entries for this library.

**Review by SSwiniarski — APPROVED** [review]

**Pull request merged**